code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -16,7 +16,9 @@
 #define KIT_DARK_LORD "darklord"
 #define KIT_WHITE_WHALE_HOLY_GRAIL "white_whale_holy_grail"
 #define KIT_MAD_SCIENTIST "mad_scientist"
+#define KIT_MAID "maid"//ORBSTATION CHANGE
 #define KIT_BEES "bee"
+#define KIT_PYRO "pyro"//ORBSTATION CHANGE
```

Comment:
id rather all of ours go together, nera the bottom or something rather than be spread out like this, (though it isnt by a lot)

Reply:
yeah I could do that didn't know if the defines order mattered *shrug*